service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyMalformedRequestsIT.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaazing.gateway.service.http.proxy;
+
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.kaazing.gateway.server.test.GatewayRule;
+import org.kaazing.gateway.server.test.config.GatewayConfiguration;
+import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
+import org.kaazing.gateway.util.feature.EarlyAccessFeatures;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+import static org.kaazing.test.util.ITUtil.createRuleChain;
+
+public class HttpProxyMalformedRequestsIT {
+
+    private final K3poRule robot = new K3poRule();
+    private final GatewayRule gateway = new GatewayRule() {{
+        // @formatter:off
+        GatewayConfiguration configuration =
+                new GatewayConfigurationBuilder()
+                    .property(EarlyAccessFeatures.HTTP_PROXY_SERVICE.getPropertyName(), "true")
+                    .service()
+                        .accept("http://localhost:8110")
+                        .connect("http://localhost:8080")
+                        .type("http.proxy")
+                        .connectOption("http.keepalive", "disabled")
+                    .done()
+                .done();
+        // @formatter:on
+        init(configuration);
+    }};
+
+    @Rule
+    public TestRule chain = createRuleChain(gateway, robot);
+
+    @Ignore // connection not accepted if "host" header is missing, was expecting 400 error
+    @Specification("http.proxy.malformed.missing.host.identifier")
+    @Test
+    public void missingHostIdentifier() throws Exception {
+        robot.finish();
+    }
+
+    @Ignore // read "GET /" instead of "read "GET http://username:password@localhost:8080/"
+    @Specification("http.proxy.malformed.user.info.on.uri")
+    @Test
+    public void userInfoOnUri() throws Exception {
+        robot.finish();
+    }
+
+    @Specification("http.proxy.malformed.invalid.request.line")
+    @Test
```

Comment:
Here the request is done to : `Host: localhost:8080`
But it should be 8110
This will not result in a response from the server.